modin/config/envvars.py

```diff
@@ -642,6 +642,20 @@ class ExperimentalNumPyAPI(EnvironmentVariable, type=bool):
     default = False
 
 
+class ExperimentalGroupbyImpl(EnvironmentVariable, type=bool):
+    """
+    Set to true to use Modin's experimental group by implementation.
+
+    Experimental groupby is implemented using a range-partitioning technique,
+    note that it may not always work better than the original Modin's TreeReduce
+    and Fold implementations. For more information visit the according section
```

Comment:
Where do we use fold in groupby?

Reply:
I thought it's an alias for full-axis functions?

Reply:
We can execute a function along full axis with `apply_full_axis`, which can change metadata. Fold is also perfomed along full axis but it has specific rules. So I would propose to correct this statement.

Reply:
done